src/theme/book.css

```diff
@@ -156,6 +156,9 @@ table thead td {
 .content img {
   max-width: 100%;
 }
+pre > .buttons {
+  z-index: 2;
```

Comment:
hi. this will get overwritten on [stylus rebuild](https://github.com/azerupi/mdBook/blob/master/CONTRIBUTING.md#making-changes-to-the-style).